src/json/json_interface.cpp

```diff
@@ -0,0 +1,101 @@
+/*******************************************************************\
+
+Module: JSON Commandline Interface
+
+Author: Peter Schrammel
+
+\*******************************************************************/
+
+/// \file
+/// JSON Commandline Interface
+
+#include "json_interface.h"
+
+#include <iostream>
```

Comment:
Nit pick: I'd prefer for standard-library includes to come last, see also some recent exchange that @kroening and I had; the idea is that any included header file should parse even without additional includes being present. So do the most minimal ones (the ones from `util`) first, and then by increasing complexity.